extension/js/common/core/crypto/pgp/openpgp-key.ts

```diff
@@ -19,7 +19,8 @@ export class OpenPGPKey {
   }
 
   public static parseMany = async (text: string): Promise<Key[]> => {
-    const result = await opgp.key.readArmored(text);
+    const result = await opgp?.key?.readArmored(text);
```

Comment:
This looks definitely suspect - if there's opgp missing, it needs to be added somewhere as a `<script>` tag

Reply:
The same as above.